cetmix_tower_server/models/cx_tower_variable_value.py

```diff
@@ -144,6 +145,15 @@ def _onchange_variable_id(self):
             else:
                 rec.option_id = None
 
+    @api.onchange("variable_id")
+    def _onchange_variable_id_set_access_level(self):
```

Comment:
You should use "compute" with "readonly=False" instead of "onchange" wherever it's possible.